tests/installation/addon_products_sle.pm

```diff
@@ -51,7 +51,7 @@ sub run {
             # might involve some network lookup of products, licenses, etc.
             assert_screen 'addon-products', 90;
             send_key "tab";                                                     # select addon-products-$addon
-            wait_still_screen 2;
+            wait_still_screen 10;
```

Comment:
In my local setup list is shown as empty first, then gets to initializing and then proper content is shown. Otherwise, start navigating in menu without a change to reach the list